### PR TITLE
Removed P2P explorer from codebase

### PR DESCRIPTION
**Pull request by Kyras**

### CODE REVIEW

File: light_node/etc/tezedge/tezedge.config [hunk 1]
```diff
@@ -72,7 +72,7 @@
 
 # <Optional> A peer to bootstrap the network from. Peers are delimited by a colon. Format: IP1:PORT1,IP2:PORT2,IP3:PORT3
 # --peers <IP:PORT>
-# --peers=
+# --peers=51.15.81.27:9732
```
Comment (bkontur) [inline light_node/etc/tezedge/tezedge.config:75]: @Kyras 
what is this IP for?

**Comment by bkontur** [comment]
@Kyras 
Can you, remove from configuration `store-p2p-messages` and all of its usage, please?

File: light_node/etc/tezedge/tezedge.config (hunk 1 above)
Comment (Kyras) [reply light_node/etc/tezedge/tezedge.config:75]: It's one of default addresses used when node is started without `--peers` argument. It is meant to be as an example in for this argument, but I will remove it for sake of consistency, as all other optional argument does not have any "sampe" value in the commented template.

Comment (Kyras) [reply light_node/etc/tezedge/tezedge.config]: Removed in 3721a55

**Comment by Kyras** [comment]
Additionally removed `--store-p2p-messages` and all mentions of it.

**Pull request merged**